auth/services/irma/irmaconfig.go

```diff
@@ -0,0 +1,133 @@
+package irma
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+
+	"github.com/nuts-foundation/nuts-node/auth/logging"
+
+	"github.com/pkg/errors"
+	irma "github.com/privacybydesign/irmago"
+	"github.com/privacybydesign/irmago/server"
+	"github.com/privacybydesign/irmago/server/irmaserver"
+)
+
+// The location the irma webserver will mount
+const IrmaMountPath = "/auth/irmaclient"
+
+// create a singleton irma config
+var _irmaConfig *irma.Configuration
+var configOnce = new(sync.Once)
```

Comment:
create an issue for removing these `syncs`

Reply:
https://github.com/nuts-foundation/nuts-node/issues/75